src/median.js

```diff
@@ -1,5 +1,31 @@
-export function median(data) {
+/* 
+ * Returns the meadian x value from a histogram of sorted abscissa
+ * @param {object} data - x sorted abscissa by incresing order and y the number of occurrence of each x value
+ * @returns {number} xmedian - the median of x values 
```

Comment:
In the returns you don't need to specify the variable name.
```suggestion
 * @returns {number}  the median of x values 
```